### ISSUE TITLE
Enable/Disable demo files

### ISSUE THREAD

**Issue by kaffeeringe** [nextcloud/server#46308]
Every time you add a new user, they get a bunch of files put into their personal folder. That is not too practical. I always have to tell them to delete those files and folders...

It would be cool to be able to disable this feature or disable it when I add a new user.

**Comment by kesselb** [comment]
https://github.com/nextcloud/server/blob/7e663301ff9461f3a4abc06c3ee80651504bfbe5/config/config.sample.php#L289-L299

**Comment by kaffeeringe** [comment]
Thank you. I knew of this possibility. But I have to talk to my hoster about that. It would be great, if this was a switch in the administration. Is it really so helpful to copy these files into every new users accounts? It doesn't feel like that. Can you please evaluate this feature?

**Comment by kesselb** [comment]
cc @szaimen @nimishavijay

**Comment by szaimen** [comment]
cc @nextcloud/designers

**Comment by marcoambrosini** [comment]
I don't think it's worth complicating things with a setting for this, but this is actually a good opportunity to think about whether we should just remove the demo files and replace them with an informative "empty-content" when there's nothing in the folder.
I think we should!

**Comment by jancborchardt** [comment]
> It would be cool, if the files were there for the first user. So if someone installs or books a new nextcloud they find the demo content to test.

Yes, as said the setting would be on by default, so by definition the first person / admin would get the example files. :)

> But for every newly created user it should be opt-in, so it's not so much of a hassle to remove the files, when you set up your organisation.

What I proposed is a global setting, but maybe instead what makes sense is just a checkbox in the account creation dialog "Create example files". Then it’s where people need it, and it can also easier be turned on and of on when needed, and the last picked setting can be remembered.

**Comment by kaffeeringe** [comment]
I can live with that! Thank you!

**Comment by MikeWilliams-UK** [comment]
Does anyone know how the system would cope (when a user first logs in) if I deleted the indicated folders?

<img width="313" height="255" alt="Image" src="https://github.com/user-attachments/assets/b0e0b792-4c9e-48de-8467-aaa179a556ec" />

At the moment I would assume that I have to remember to delete these folders after each update...
